### PR TITLE
GitOps Operator | Adding periodic jobs

### PR DESCRIPTION
**Pull request by varshab1210**

### CODE REVIEW

File: ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: redhat-developer
+    repo: gitops-operator
+  labels:
+    ci-operator.openshift.io/variant: v4.12
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-developer-gitops-operator-master-v4.12-periodic-kuttl-parallel
```
Comment (anandrkskd) [inline ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml:14]: I think we should add notification for our channel if the tests failed, WDYT?

**Review by anandrkskd — APPROVED** [review]
Changes looks good, there is only one thing that I would like to add!

File: ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml (hunk 1 above)
Comment (varshab1210) [reply ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml:14]: Sounds good! I'll take a look on how I could implement it

**Comment by varshab1210** [comment]
/pj-rehearse pull-ci-redhat-developer-gitops-operator-master-v4.14-kuttl-sequential-periodic

File: ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+  reporter_config:
+    slack:
+      channel: '#gitops-dev-qe'
```
Comment (anandrkskd) [inline ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml:17]: Are we okay using gitops-dev-qe?

Comment (varshab1210) [reply ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml]: Sorry! I meant to add **#gitops-test-notification**

File: ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-presubmits.yaml [hunk 3]
```diff
@@ -972,3 +1046,77 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )v4.14-kuttl-sequential,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (anandrkskd) [inline ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-presubmits.yaml:1050]: you might need to change this?

**Pull request merged**